mavisp/core.py

```diff
@@ -244,6 +252,14 @@ def ingest(self, stop_at='critical'):
                     mavisp_criticals.append(MAVISpCriticalError("Uniprot AC was not found in the metadata file"))
                 if refseq_id is None:
                     mavisp_criticals.append(MAVISpCriticalError("RefSeq ID was not found in the metadata file"))
+                if review_status is None:
+                    mavisp_criticals.append(MAVISpCriticalError("Review status was not found in the metadata file"))
+                else:
+                    if str.isdigit(review_status) is False:
```

Comment:
the better (most robust) way to do comparison with boolean is just to use `if condition`:

```if not str.isidigt(...):
```